back/myapp/scripts/users.js

```diff
@@ -0,0 +1,28 @@
+var { USER_ERRORS } = require("../constants/errors");
+var { loginConfig } = require("../constants/config");
+var { LOGIN } = require("../constants/errors");
+var { User } = require("../models/User");
+var { getPathToImages, updateImagePaths } = require("./utils")
+var jwt = require("jsonwebtoken");
+var { dispatchError } = require("./errorHandlers/common")
+
+
+function getUserData(req, res, next){
+    jwt.verify(req.token, loginConfig.secretKey, (err, authData) => {
+      if(err) {
+          dispatchError(res,next,LOGIN.UNAUTHORIZED,403);
+      }
+      else {
+        var { id } = req.body;
+        User.findById(id, "firstname surname email avatar minAvatar birthdate parrentname", (err, result) => {
```

Comment:
the same request with the same field list I've seen before, probably you should keep the list of fields somewhere

Reply:
i think that in future this requests will have different field list